### ISSUE TITLE
keep this repo?

### ISSUE THREAD

**Issue by matthewberryman** [urbanriskmap/petajakarta.org#6]

**Comment by talltom** [comment]
yes; long term I would like to import some of the static content from original petajakarta.org (white paper etc.) back up under the petajakarta.org domain. This is low priority during the 2017 monsoon season though.

**Comment by matthewberryman** [comment]
@talltom that content is all under the [smart-facility/petajakarta-web](https://github.com/smart-facility/petajakarta-web) repo, not in this repo, with the exception of a duplicate copy of the web page icon. This repo is just the static content for the redirection page. Deleting this repo therefore won't stop you doing what you want to do.

**Comment by talltom** [comment]
@matthewberryman OK. Let's leave as is until monsoon in Jakarta is over, then work on updates to smart-facility/petajakarta-web to link to petabencana.id?

Don't want to confuse users during the monsoon, so redirect is great for now.

**Comment by talltom** [comment]
p.s. a little beyond scope, but when we do get there we should investigate whether we could host PetaJakarta.org static pages and materials via github pages to save ongoing server costs (i.e. future proof against completion of projects).

**Comment by matthewberryman** [comment]
I would say best to delete this repo, leaving the static content on the site (not to self: disconnect jenkins build), and then fork the petajakarta-web repo and optionally rename it to petajakarta.org. That can be done at any time without affecting web content at petajakarta.org (the site) but happy to defer if you like.

I would argue it's better to use s3+cloudfront; you can host a small static site for a few dollars a year and take advantage of AWS' CDN (incl. IPv6 and http/2).

**Comment by talltom** [comment]
Now we have the static branch over at https://github.com/smart-facility/petajakarta-web/commits/static we can get rid of this holding page.

**Comment by matthewberryman** [comment]
@talltom will do
